websites/R/Radio Garden/presence.ts

```diff
@@ -2,18 +2,48 @@ const presence = new Presence({
   clientId: "687070418804408445" //The client ID of the Application created at https://discordapp.com/developers/applications
 });
 
-presence.on("UpdateData", async () => {
-  if (document.location.pathname.startsWith("/listen/")) {
-    const elapsed = Math.floor(Date.now() / 1000);
-    //var container = document.querySelector(".channel-list")
-    //console.log(document.querySelector("div.mod-active").innerText)
-    const presenceData: PresenceData = {
-      details: `${document.querySelector("div.mod-active").textContent}`,
-      state: `${document.querySelector(".location-info-location").textContent}`,
-      largeImageKey: "bigglobe",
-      startTimestamp: elapsed
-    };
+let details: HTMLInputElement,
+state: HTMLInputElement;
 
-    presence.setActivity(presenceData);
-  }
-});
+presence.on("UpdateData", async () => {
+	if(!document.title.includes("Explore live radio by rotating")){
+		const elapsed =  Math.floor(Date.now() / 1000);
+		const presenceData: PresenceData = {
+//			details: `${document.querySelector(".ChannelTitle_title__2QQj5").innerText}`,
+//			state: `${document.querySelector(".ChannelTitle_subtitle__DZ_ZQ").innerText}`,
```

Comment:
why commented?

Reply:
Updated and uncommented, also fixes the obscure behaviour from comment above